test/unit/pipeline/FetchUnitTest.cc

```diff
@@ -417,7 +418,7 @@ TEST_P(PipelineFetchUnitTest, idleLoopBufferDueToNotTakenBoundary) {
 
   // Set the first expectation from the predictor to be true so a loop body will
   // be detected
-  EXPECT_CALL(predictor, predict(_, _, _))
```

Comment:
I think we should be doing `predict(_, _, _, true)` given we are in a loop and expect the value to be true.

Reply:
So, here I think that we are filling the loop buffer, rather than using the loop buffer to supply the instructions.  Therefore, it should be predict(_, _, _, false).  

I agree that this means that the name of the new boolean value isLoop might be too vague.  How about something like loopBufferIsSupplying - is that too verbose?